packages/pointer_interceptor/pointer_interceptor_web/example/integration_test/widget_test.dart

```diff
@@ -4,22 +4,21 @@
 
 // ignore_for_file: avoid_print
 
-import 'dart:html' as html;
-
 // Imports the Flutter Driver API.
 import 'package:flutter/src/widgets/framework.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:integration_test/integration_test.dart';
 
 import 'package:pointer_interceptor_web_example/main.dart' as app;
 
+import 'package:web/web.dart' as web;
```

Comment:
Nit: shouldn't this be up with the other packages? Usually we just have three blocks: `dart:`, `package:`, and path-based.

Reply:
Yep, will tidy the imports and make the `onElementCreated` a tear-off as suggested by @mdebbar above.